[2023-11-27 09:36:32,222] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_province_daily manual__2023-11-27T09:22:18.570735+00:00 [queued]>
[2023-11-27 09:36:32,231] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_fact_province_daily manual__2023-11-27T09:22:18.570735+00:00 [queued]>
[2023-11-27 09:36:32,231] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 09:36:32,231] {taskinstance.py:1357} INFO - Starting attempt 4 of 6
[2023-11-27 09:36:32,231] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 09:36:32,245] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_fact_province_daily> on 2023-11-27 09:22:18.570735+00:00
[2023-11-27 09:36:32,249] {standard_task_runner.py:52} INFO - Started process 933 to run task
[2023-11-27 09:36:32,252] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'final_project_data_covid_etl', 'create_fact_province_daily', 'manual__2023-11-27T09:22:18.570735+00:00', '--job-id', '401', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmprxfr1vbo', '--error-file', '/tmp/tmp8bfex79c']
[2023-11-27 09:36:32,253] {standard_task_runner.py:80} INFO - Job 401: Subtask create_fact_province_daily
[2023-11-27 09:36:32,303] {task_command.py:369} INFO - Running <TaskInstance: final_project_data_covid_etl.create_fact_province_daily manual__2023-11-27T09:22:18.570735+00:00 [running]> on host f38c0c12695c
[2023-11-27 09:36:32,381] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ricky
AIRFLOW_CTX_DAG_ID=final_project_data_covid_etl
AIRFLOW_CTX_TASK_ID=create_fact_province_daily
AIRFLOW_CTX_EXECUTION_DATE=2023-11-27T09:22:18.570735+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-27T09:22:18.570735+00:00
[2023-11-27 09:36:32,382] {python.py:173} INFO - Done. Returned value was: None
[2023-11-27 09:36:32,395] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=final_project_data_covid_etl, task_id=create_fact_province_daily, execution_date=20231127T092218, start_date=20231127T093632, end_date=20231127T093632
[2023-11-27 09:36:32,424] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-27 09:36:32,473] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
